capacitor.config.json

```diff
@@ -20,5 +29,6 @@
       }
     }
   },
-  "cordova": {}
+  "cordova": {},
+  "linuxAndroidStudioPath": "/opt/android-studio-4.1/android-studio/bin/studio.sh"
```

Comment:
@Abhishek-kumar09 I think this should be removed, right ?
I added this because on running npx cap open android, I was getting an error saying to add this path. But I think it will be different on different systems.